autokeras/image_supervised.py

```diff
@@ -365,3 +374,32 @@ def transform_y(self, y_train):
 
     def inverse_transform_y(self, output):
         return output.flatten()
+
+class PortableImageClassifer(PortableClass):
+	def predict(self, x_test):
+        """Return predict results for the testing data.
+
+        Args:
+            x_test: An instance of numpy.ndarray containing the testing data.
+
+        Returns:
+            A numpy.ndarray containing the results.
+        """
+        #DataTransformer needs to be fixed
+        DataTransformer(x_test, augment=Constant.DATA_AUGMENTATION)
```

Comment:
Instead of using a new DataTransformer, you should copy the one from ImageSupervised class.
And the copy the y_encoder as well.
Copy them by adding more parameters to the constructor.